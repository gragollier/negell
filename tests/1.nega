int g;

int fac(int a) {
    if (a <= 1) {
        return 1;
    }
    else {
        return a * fac(a - 1);
    }
}

int test() {
    output g;
    g = 2;
    return g;
}

int main() {
    int a;
    input a;
    output "You said: ";
    output a;

    a = 2;
    a = a - 1;

    output "Factorial of 5 is ";
    output fac(5);

    output test();
    output a / 1;

    if (!a) {
        output 69;
    } else {
        output 420;
    }
    a = 0;
    while (a <= 10) {
        output a;
        a = a - -1;
    }
    output a;

    return 0;
}