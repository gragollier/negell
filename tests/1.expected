You said: 
123
Factorial of 5 is 
120
0
2
1
420
0
1
2
3
4
5
6
7
8
9
10
11
